models/zeroex/bnb/zeroex_bnb_api_fills.sql

```diff
@@ -347,6 +347,7 @@ uni_v2_swap as (
 
 
 )
+-- TODO: this is really slow, need to optimize
```

Comment:
this will trigger full refresh, but the plan for this PR will be to turn off modified anyway, so we can let it be